RecommendationEngineServer/RecommendationEngineServer.Logic/Admin/AdminService.cs

```diff
@@ -0,0 +1,242 @@
+﻿using RecommendationEngineServer.Common;
+using RecommendationEngineServer.Common.DTO;
+using RecommendationEngineServer.Common.Enum;
+using RecommendationEngineServer.Common.Exceptions;
+using RecommendationEngineServer.DAL.Models;
+using RecommendationEngineServer.DAL.UnitOfWork;
+
+namespace RecommendationEngineServer.Service.Admin
+{
+    //public class AdminService : IAdminService
```

Comment:
why keep the commented code? If you want to keep code backups. you should use branches instead of maintaining huge chunks of commented code